packages/@aws-cdk/aws-cloud9/test/cloud9.environment.test.ts

```diff
@@ -0,0 +1,69 @@
+import { expect as expectCDK, haveResource } from '@aws-cdk/assert';
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as cdk from '@aws-cdk/core';
+import * as cloud9 from '../lib';
+
+let stack = new cdk.Stack();
```

Comment:
Probably this would be simpler:

```ts
let stack: cdk.Stack;
let vpc: ec2.IVpc;
```

and then initialize them in the `beforeEach` like you do.